tests/test_utilities.py

```diff
@@ -403,24 +426,23 @@ def test_generate_predictions_conflicting_params():
 
 
 def test_generate_predictions():
-    for (use_feature_hashing,
-         use_threshold,
-         test_on_subset,
-         all_probabilities) in product([True, False], [True, False],
-                                       [True, False], [True, False]):
+    for (use_feature_hashing, use_threshold, test_on_subset, all_probabilities,
+         string_labels) in product(*[[True, False]] * 5):
+
         if use_threshold and all_probabilities:
             continue
         yield (check_generate_predictions, use_feature_hashing,
-               use_threshold, test_on_subset, all_probabilities)
+               use_threshold, test_on_subset, all_probabilities,
+               string_labels)
 
 
 def test_generate_predictions_file_header():
 
-    for (use_threshold, all_probabilities) in ([True, False], [False, True]):
+    for (use_threshold, all_probabilities, string_labels) in product(*[[True, False]] * 3):
```

Comment:
I would actually argue for having the expanded form of the list rather than using `product` since that makes it easier for even a novice contributor to figure out what's going on.